scenes/quests/lore_quests/quest_001/1_music_puzzle/components/bonfire_sign/bonfire_sign.tscn

```diff
@@ -0,0 +1,182 @@
+"texture": ExtResource("7_pf82u")
+}],
+"loop": true,
+"name": &"forward_off",
+"speed": 5.0
+}, {
+"frames": [{
+"duration": 1.0,
+"texture": ExtResource("3_kbib7")
+}],
+"loop": true,
+"name": &"forward_on",
+"speed": 5.0
+}, {
+"frames": [{
+"duration": 1.0,
+"texture": ExtResource("9_ypj2g")
+}],
+"loop": true,
+"name": &"forward_up_off",
+"speed": 5.0
+}, {
+"frames": [{
+"duration": 1.0,
+"texture": ExtResource("10_lgc81")
+}],
+"loop": true,
+"name": &"forward_up_on",
+"speed": 5.0
+}]
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_1u12m"]
+atlas = ExtResource("4_gye5d")
+region = Rect2(0, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_7vc6p"]
+atlas = ExtResource("4_gye5d")
+region = Rect2(128, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_te87b"]
+atlas = ExtResource("4_gye5d")
+region = Rect2(256, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_n2cce"]
+atlas = ExtResource("4_gye5d")
+region = Rect2(384, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_wygyr"]
+atlas = ExtResource("4_gye5d")
+region = Rect2(512, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_ehqd8"]
+atlas = ExtResource("4_gye5d")
+region = Rect2(640, 0, 128, 128)
+
+[sub_resource type="AtlasTexture" id="AtlasTexture_e4kms"]
+atlas = ExtResource("4_gye5d")
+region = Rect2(768, 0, 128, 128)
+
+[sub_resource type="SpriteFrames" id="SpriteFrames_p0tgm"]
+animations = [{
+"frames": [{
+"duration": 0.5,
+"texture": SubResource("AtlasTexture_1u12m")
+}, {
+"duration": 0.5,
+"texture": SubResource("AtlasTexture_7vc6p")
+}, {
+"duration": 0.5,
+"texture": SubResource("AtlasTexture_te87b")
+}, {
+"duration": 0.5,
+"texture": SubResource("AtlasTexture_n2cce")
+}, {
+"duration": 0.5,
+"texture": SubResource("AtlasTexture_wygyr")
+}, {
+"duration": 0.5,
+"texture": SubResource("AtlasTexture_ehqd8")
+}, {
+"duration": 0.5,
+"texture": SubResource("AtlasTexture_e4kms")
+}],
+"loop": true,
+"name": &"burning",
+"speed": 5.0
+}, {
+"frames": [],
+"loop": true,
+"name": &"default",
+"speed": 5.0
+}]
+
+[sub_resource type="CircleShape2D" id="CircleShape2D_kbib7"]
+
+[sub_resource type="CircleShape2D" id="CircleShape2D_gye5d"]
+radius = 34.3657
+
+[node name="BonfireSign" type="StaticBody2D"]
+texture_filter = 1
+collision_layer = 16
+collision_mask = 0
+script = ExtResource("1_4621y")
+
+[node name="BaseSprite" type="Sprite2D" parent="."]
+position = Vector2(0, -32)
+texture = ExtResource("2_1po6h")
+
+[node name="SignSprite" type="AnimatedSprite2D" parent="BaseSprite"]
+unique_name_in_owner = true
+sprite_frames = SubResource("SpriteFrames_mtebg")
+animation = &"forward_off"
+
+[node name="Fire" type="AnimatedSprite2D" parent="."]
+unique_name_in_owner = true
+position = Vector2(0, -88)
+sprite_frames = SubResource("SpriteFrames_p0tgm")
+
+[node name="CollisionShape2D" type="CollisionShape2D" parent="."]
+scale = Vector2(1.4, 1)
+shape = SubResource("CircleShape2D_kbib7")
```

Comment:
I think this collision area might be a bit small – I can walk through the edge of the sign. But I couldn't get stuck, so maybe it's OK.

Reply:
The idea is that only the base collides. You can go "behind" the sign, and that's OK. Do you think I should change that?